gateway/crates/gateway-binary/tests/integration_tests.rs

```diff
@@ -654,3 +666,20 @@ fn hybrid_graph() {
         "###);
     });
 }
+
+#[test]
+fn health() {
+    let config = "";
+
+    let schema = "type NotValid { this: DoesNotExist! }";
+
+    with_static_server(config, &schema, None, None, |client| async move {
+        let (_, result): (_, serde_json::Value) = client.get("/health").await;
+
+        insta::assert_json_snapshot!(&result, @r###"
+        {
+          "status": "healthy"
+        }
+        "###);
+    });
+}
```

Comment:
I'd have liked to have a test for the unhealthy status, but I couldn't make it happen — seems we fail early before there's a chance to become unhealthy, which is a good thing.